src/Widgets/AppEntry.vala

```diff
@@ -122,17 +145,30 @@ public class Slingshot.Widgets.AppEntry : Gtk.Button {
         image = new Gtk.Image ();
         image.gicon = app.icon;
         image.pixel_size = ICON_SIZE;
-        image.margin_top = 12;
+        image.margin_top = 9;
+        image.margin_end = 6;
+        image.margin_start = 6;
 
-        count_image = new Gtk.Image ();
-        count_image.no_show_all = true;
-        count_image.visible = false;
-        count_image.margin_start = ICON_SIZE - SURFACE_SIZE;
-        count_image.margin_bottom = ICON_SIZE - SURFACE_SIZE;
```

Comment:
I suppose that the provider stuff should go into `static construct` rather than putting it into here? It doesn't use anything from the instance and is redundant for next creations of `AppEntry`. I could be wrong here though.